src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -26,6 +26,10 @@ public SchedulerConf() {
     setConf(configuration);
   }
 
+  public String getExecutorPath() {
+    return getConf().get("mesos.hdfs.executor.config.path", "..");
```

Comment:
Yes, this is the default when untaring our build and runs successfully. I wanted to make a simple user experience, although this can of course be configured.